packages/plugin-ext/src/main/browser/debug/plugin-debug-adapter-contribution.ts

```diff
@@ -14,7 +14,7 @@
  * SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
  ********************************************************************************/
 
-import { DebugExt, } from '../../../common/plugin-api-rpc';
+import { DebugExt} from '../../../common/plugin-api-rpc';
```

Comment:
I don't know why but I thought eslint would complain for missing spaces between curly brackets?